modules/javafx.controls/src/main/java/javafx/scene/control/Labeled.java

```diff
@@ -818,6 +818,48 @@ public final BooleanProperty mnemonicParsingProperty() {
         return mnemonicParsing;
     }
 
+    /**
+     * Truncated read-only property indicates whether the text has been truncated
+     * when it cannot fit into the available width.
+     * <p>
+     * When truncated, the {@link #ellipsisStringProperty() ellipsis string}
+     * gets inserted in the place dictated by the
+     * {@link #textOverrun} property.
+     *
+     * @since 23
+     * @defaultValue false
+     */
+    private ObservableBooleanValue truncated;
```

Comment:
The property name should probably be `textTruncated` to be in line with `textOverrun`, `textFill`, `wrapText`...

Otherwise it could look like the labeled is truncated.